packages/mpl-multitab.yml

```
@@ -0,0 +1,6 @@
+repo: astromancer/mpl-multitab
+section: gui applications
+description: Tabbed figure manager for matplotlib using pyQt.
+site: https://github.com/astromancer/mpl-multitab
+keywords: [matplotlib, GUI, figure manager, tabs, tabbed, pyside, pyqt] # optional
```

```suggestion
keywords: [GUI, figure manager, tabs, tabbed, pyside, pyqt]
```
"matplotlib" seems redundant here, considering the context. Apart from that it looks good! Thanks!